Overhaul of Adapter Development Contributing Docs (#1727)
fishtown-analytics/docs.getdbt.com: website/docs/docs/contributing/adapter-development/4-documenting-a-new-adapter.md

---
title: "Documenting a new adapter"
id: "4-documenting-a-new-adapter"
---

> If you've already [built](2-building-a-new-adapter), and [tested](3-testing-a-new-adapter) your adapter, it's time to document it so the dbt community will know that it exists and how to use it!

Reviewer: ```suggestion
If you've already [built](2-building-a-new-adapter), and [tested](3-testing-a-new-adapter) your adapter, it's time to document it so the dbt community will know that it exists and how to use it.
```

## Making your adapter available

Many community members maintain their adapter plugins under open source licenses. If you're interested in doing this, we recommend:

> - Hosting on a public git provider (e.g. GitHub, GitLab)

Reviewer: ```suggestion
- Hosting on a public git provider (for example, GitHub or Gitlab)
```
